style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background: #231a59;
+  min-height: 100vh;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  color: white;
+  margin: 1rem;
+  margin-top: 3rem;
+}
+
+.hover-button:hover {
+  background: blueviolet;
+}
+
+header h2 {
+  margin: 0.5rem 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  letter-spacing: 0.18rem;
+  font-size: 2rem;
+  font-weight: lighter;
+}
+
+h1 {
+  padding: 1rem;
+  color: rgb(131, 90, 255);
+  text-align: center;
+  font-size: 3rem;
+  margin-top: 1rem;
+}
+
+section h2 {
+  text-align: center;
+  font-size: 3.5rem;
+  font-weight: lighter;
+  margin-top: 2rem;
+  color: rgb(223, 223, 223);
+}
+
+section img {
+  position: absolute;
+  z-index: 1;
+}
+
+section {
+  color: white;
+}
+
+p {
```

Comment:
[Optional]

- Instead of styling using tag, it's better to use `class` in HTML and then style accordingly. This will help you in writing clean code, using DRY (Do not repeat yourself) principle.

Suppose I have two use one property `display: flex`  many times in html. then it's always better to make one class for flex and use them in different sections, rather than writing display:flex for each tag.

ex:
```
Bad way 

section,
header,
footer,
p {
display: flex;
}


Good way

.flex {
display: flex;
}

and then use this class in HTML like this:
<HTML FILE>

<section class="flex">
<p class="flex"> random text </p>
</section>
<footer class="flex">
.....
</footer>

```

Reply:
Thanks for the suggestion I already implemented it.